plugins/inputs/elasticsearch/README.md

```diff
@@ -33,6 +33,10 @@ or [cluster-stats](https://www.elastic.co/guide/en/elasticsearch/reference/curre
   ## Master node.
   cluster_stats = false
 
+  ## Set individual_stats to a list of sub-stats you are interested in
+  ## default is all stats
+  # individual_stats = ["jvm"]
```

Comment:
I think it will be difficult to know what these values apply to as is.  Would it help if we call this like `node_stats = ["jvm"]`?  We should show the default in the commented out line, and it will also help if we list the current valid options:
```
+  ## Node stats are the statistic categories to gather for each node.  Valid
+  ## options are `indices`, `os`, `process`, `jvm`, `transport`, `http`, `fs`,
+  ## `breaker` and `thread_pool`.  If empty, all categories are gathered.
+  # nodes_stats = []
```